src/pages/Thanks/ThanksPage.vue

```diff
@@ -9,6 +9,9 @@
 				:show-daf-thanks="showDafThanks"
 			/>
 		</template>
+		<template v-if="activeView === SINGLE_VERSION_VIEW">
```

Comment:
Since this template will render first (see other comment), move this above donation for clarity.